src/main/java/pl/allegro/tech/embeddedelasticsearch/InstanceDescription.java

```diff
@@ -1,19 +1,30 @@
 package pl.allegro.tech.embeddedelasticsearch;
 
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Map;
+
 class InstanceDescription {
```

Comment:
This class changed it's role a little. Maybe we should rename it to InstanceSettings?